scratch-js/Sprite.mjs

```diff
@@ -609,4 +705,13 @@ export class Stage extends SpriteBase {
     // For obsolete counter blocks.
     this.__counter = 0;
   }
+
+  get fence() {
+    return {
+      left: -this.width / 2,
+      right: this.width / 2,
+      top: this.height / 2,
+      bottom: -this.height / 2
+    };
+  }
```

Comment:
Could you move this to the constructor to avoid creating a new object every time `keepInFence` is called?

Reply:
Thanks, I'll do that